endoreg_db/utils/video/ffmpeg_wrapper.py

```
@@ -13,6 +15,67 @@
+    # 2) Django settings overrides (if Django is configured)
+    try:
+        from django.conf import settings  # type: ignore
+
+        env_candidates.extend(
+            getattr(settings, attr)
+            for attr in ("FFMPEG_EXECUTABLE", "FFMPEG_BINARY", "FFMPEG_PATH")
+            if hasattr(settings, attr)
+        )
+    except Exception:
+        # Django might not be configured for every consumer
+        pass
```

Ruff (0.14.0)

`try`-`except`-`pass` detected, consider logging the exception

(S110)